packages/fluentui/docs/src/examples/components/Datepicker/Usage/DatepickerExampleSimple.tsx

```diff
@@ -0,0 +1,6 @@
+import { Datepicker } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerExampleSimple = () => <Datepicker />;
```

Comment:
Usually, by convention the default example's name should be just `ComponentNameExample`, so I would suggest renaming this context and the files to `DatePickerExample`

Reply:
Good catch, changed.